netbox/tenancy/filtersets.py

```diff
@@ -60,17 +65,17 @@ class Meta:
 
 
 class ContactFilterSet(NetBoxModelFilterSet):
-    group_id = TreeNodeMultipleChoiceFilter(
+    contact_group_id = TreeNodeMultipleChoiceFilter(
```

Comment:
Leaving these as `group_id` and `group` seems to work fine AFAICT. We should avoid renaming them if possible.